src/components/bulk-email-tool/bulk-email-form/test/BulkEmailForm.test.jsx

```diff
@@ -28,32 +30,24 @@ describe('bulk-email-form', () => {
     expect(warning).toBeTruthy();
   });
   test('Prevent form POST if invalid', async () => {
-    await act(async () => {
```

Comment:
Should I avoid the `await act` pieces too?

Reply:
Apparently `@test-library/react` will automatically wrap certain methods from the library in `act()`, but I was having issues with state updates not working properly originally. I used it as a band-aid for making sure updates happened. Then my tests broke for an unrelated reason and I refactored them again. I found the act unnecessary this time around. Your mileage may vary, but if it works I don't see a need to remove it. (I'm still learning `test-library` but the standard used to be to just wrap everything in act)

Reply:
Thanks! Good to know that it is something I can try if I see any weird behavior in my tests.